docs/usage/session.rst

```diff
@@ -0,0 +1,59 @@
+.. _session:
+
+==================
+Session management
+==================
+
+Zyte API provides powerful session APIs:
+
+-   Client-managed sessions give you full control over session management.
+
+-   :ref:`Server-managed sessions <zyte-api-session-contexts>` let Zyte API
+    handle session management for you.
+
+When using scrapy-zyte-api, you can use these session APIs through the
+corresponding Zyte API fields (:http:`request:session`,
+:http:`request:sessionContext`).
+
+However, scrapy-zyte-api also provides :ref:`its own session API
+<plugin-sessions>`.
+
+.. _plugin-sessions:
+
+scrapy-zyte-api’s session API
+=============================
+
+scrapy-zyte-api’s session API offers a higher-level API on top of
+client-managed sessions, to enjoy some of the advantages of client-managed
+sessions (e.g. expiring specific sessions) while removing some of their
+drawbacks (management overhead).
+
+To use scrapy-zyte-api’s session API, define
+:setting:`ZYTE_API_SESSION_PARAMS` or :setting:`ZYTE_API_SESSION_CHECKER`.
+
+Often it makes sense to define both. For example:
+
+.. code-block:: python
+    :caption: settings.py
+
+    ZYTE_API_SESSION_PARAMS = {
+        "actions": [
+            {
+                "action": "setLocation",
+                "address": {"postalCode": "04662"},
+            }
+        ]
+    }
+
+
+    class MySessionChecker:
+
+        def check(self, request, response):
```

Comment:
Is this the method that's going to be used to check the session's validity?

Reply:
Yes.

Reply:
Yes, it is :-) I'd rather have a more meaningful name, may be `check_session` and a return type would help also.

Reply:
https://github.com/scrapy-plugins/scrapy-zyte-api/pull/193/commits/332c15d1ad169941e39eb3d3d954d758699d4bf8